test/common/config/grpc_mux_impl_test.cc

```diff
@@ -341,31 +344,17 @@ TEST_F(GrpcMuxImplTestWithMockTimeSystem, TooManyRequests) {
   };
 
   auto foo_sub = grpc_mux_->subscribe("foo", {"x"}, callbacks_);
+
   expectSendMessage("foo", {"x"}, "");
   grpc_mux_->start();
+  std::cout << "after on start..."
```

Comment:
I don't think you meant to leave that here :)

Reply:
Thanks :-)